error.go

```diff
@@ -123,6 +123,11 @@ func IsPreconditionFailed(err error) bool {
 	return IsArangoErrorWithCode(err, 412) || IsArangoErrorWithErrorNum(err, 1200, 1210)
 }
 
+// IsNoLeader returns true if the given error is an ArangoError with code 503 error number 1496.
```

Comment:
Comment indicates "and" and code does "||"?

Reply:
changed the code